### DISCUSSION TITLE
Új belépő.

### DISCUSSION THREAD

**Discussion by balsistv** [community/community#82190] [New to GitHub]
Új belépő vagyok és nehezen ismerem ki magam a sok szakkifejezés között, mert ezekkel még nem találkoztam. Azért léptem be ide, mert két crypto tárcát szeretnék egymással kapcsolatba hozni és nem találom az erre vonatkozó lehetőséget. Nagyon megköszönném ha valaki tudna ebben a segítségemre lenni akár emailben is: balsistv@gmail.com. Üdvözlettel: István

**Comment by Imadnajam** [comment]
,

**Comment by Akash1134** [comment]
Hi @balsistv , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
